main.cpp

```
@@ -1,21 +1,32 @@
+
 int main() {
-  std::cout << std::unitbuf;
-  std::cout << "cppchess_engine version " << BUILD_VERSION << '\n';
-  options.add("Move Overhead", Option(10, 0, 1000));
-  options.add("Hash", Option(16, 1, 1 << 25,
-                             [](const Option &o) { return std::nullopt; }));
+    std::cout << std::unitbuf;
+    std::cout << "cppchess_engine version " << BUILD_VERSION << '\n';
+    options.add("Move Overhead", Option(10, 0, 1000));
+    options.add("Hash", Option(16, 1, 1 << 25, [](const Option &o) {
+                    search::tt.resize(int(o));
```

Cppcheck (2.21.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)